src/lib.rs

```diff
@@ -141,10 +141,45 @@
 //! println!("VOPRF output: {:?}", client_finalize_result.to_vec());
 //! ```
 //!
+//! ### Server Evaluation
+//!
+//! In the final step on the server side, the server runs [OprfServer::evaluate]
+//! to finish the OPRF protocol by using its private key to produce an output
+//! that is equal to the client's output.
```

Comment:
I think this text could be changed. You have might context into the motivation for adding the `Evaluate` function into the draft spec, but perhaps something along the lines of:

"Optionally, if the server has direct access to the PRF input, then it need not perform the oblivious computation and can simply run [OprfServer::evaluate] to generate an output which matches the output produced by an execution of the oblivious protocol on the same input and key."